### PR TITLE
INTYGFV-15226: Refactored creating filled in test certificates from t…

### PR DESCRIPTION
**Pull request by mww-sopra-steria**
…he testability api. Each certificate type implementation has the responsibility to fill in the certificate of a certain type and major version.

### CODE REVIEW

File: support/src/main/java/se/inera/intyg/common/support/facade/util/TestabilityToolkit.java [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+public class TestabilityToolkit {
+
+    private static final int DEFAULT_SICK_LEAVE_LENGTH = 14;
+    private static final int DEFAULT_SHORT_SICK_LEAVE_LENGTH = 4;
+
+    public static void fillCertificateWithTestData(Certificate certificate, FillType fillType,
+        TestabilityTestdataToolkit testabilityTestdataToolkit) {
+        if (FillType.MINIMAL.equals(fillType)) {
+            final var minimumValuesLisjp = testabilityTestdataToolkit.getMinimumValues();
```
Comment (ChrilleSigerhed) [inline support/src/main/java/se/inera/intyg/common/support/facade/util/TestabilityToolkit.java:43]: Should we change this to minimumValues & maximumValues?

File: fk7263/src/main/java/se/inera/intyg/common/fk7263/testability/TestDataUtil.java [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+import java.time.LocalDate;
+import se.inera.intyg.common.fk7263.model.internal.Fk7263Utlatande;
+import se.inera.intyg.common.fk7263.model.internal.PrognosBedomning;
+import se.inera.intyg.common.fk7263.model.internal.Rehabilitering;
+import se.inera.intyg.common.support.model.InternalDate;
+import se.inera.intyg.common.support.model.InternalLocalDateInterval;
+import se.inera.intyg.common.support.model.LocalDateInterval;
+
+public class TestDataUtil {
```
Comment (ChrilleSigerhed) [inline fk7263/src/main/java/se/inera/intyg/common/fk7263/testability/TestDataUtil.java:30]: Should we implement an Interface that extends decorateWithMinimalValues/decorateWithMaximumValues, similiar to getMaximalValues/getMinimalValues in TestabilityTestdataToolkit?

**Review by ChrilleSigerhed — APPROVED** [review]
Great work, I left a suggestion regarding the implementation of an interface for decorateWithValues that we discussed!

File: support/src/main/java/se/inera/intyg/common/support/facade/util/TestabilityToolkit.java (hunk 1 above)
Comment (mww-sopra-steria) [reply support/src/main/java/se/inera/intyg/common/support/facade/util/TestabilityToolkit.java]: Nice catch!

File: fk7263/src/main/java/se/inera/intyg/common/fk7263/testability/TestDataUtil.java (hunk 2 above)
Comment (mww-sopra-steria) [reply fk7263/src/main/java/se/inera/intyg/common/fk7263/testability/TestDataUtil.java]: Good suggestion! Done!

**Pull request merged**